end_to_end/end_to_end_suite_test.go

```
@@ -65,6 +65,10 @@ var _ = Describe("backup end to end integration tests", func() {
 		if err != nil {
 			Fail(fmt.Sprintf("%v", err))
 		}
+		err = exec.Command("createdb", "restoredb").Run()
+		if err != nil {
+			Fail(fmt.Sprintf("%v", err))
+		}
 		backupConn = utils.NewDBConn("testdb")
 		backupConn.Connect()
 		utils.ExecuteSQLFile(backupConn, "test_tables.sql")
```

We could also change this to be like 
``` 
connStr := []string{
"-d", "testdb",
"-f", "all_objects.sql",
"-q",
}
exec.Command("psql", connStr...)
```
below and then we can remove the `ExecuteSQLFile` function.

---

The ExecuteSQLFile function also sets `ON_ERROR_STOP=1`, which prevents the swallowing of errors. I think instead we should use this function when executing sql files so we don't accidentally commit bad syntax.